src/transformers/models/dpt/configuration_dpt.py

```diff
@@ -94,6 +99,12 @@ class DPTConfig(PretrainedConfig):
             The index that is ignored by the loss function of the semantic segmentation model.
         semantic_classifier_dropout (`float`, *optional*, defaults to 0.1):
             The dropout ratio for the semantic classification head.
+        backbone_featmap_shape (`List[int]`, *optional*, defaults to `[1, 1024, 24, 24]`):
+            Used only for the `hybrid` embedding type. The shape of the feature maps of the backbone.
+        neck_ignore_stages (`List[int]`, *optional*, defaults to `[0, 1]`):
+            Used only for the `hybrid` embedding type. The stages of the readout layers to ignore.
+        backbone_config (`Dict[str, Any]`, *optional*, defaults to `None`):
```

Comment:
I see the backbone_config must be a dict here, but I've made the backbone API such that you can directly pass a config object, like so:
```
from transformers import ResNetConfig, MaskFormerConfig, MaskFormerModel

backbone_config = ResNetConfig(out_features=["stage1", "stage2", "stage3", "stage4"])
config = MaskFormerConfig(backbone_config=backbone_config)
model = MaskFormerModel(config)
```
=> can we allow the same for DPT-hybrid? It a lot more convenient to not having to pass all attributes of the config manually in a dict

Reply:
Yes we can!
Proposed a solution in [9bc9e5a](https://github.com/huggingface/transformers/pull/20645/commits/9bc9e5a5b0a99660330289e6ca4939114b21b5af)